fog/types/Cargo.toml

```diff
@@ -12,6 +12,7 @@ mc-transaction-core = { path = "../../transaction/core" }
 mc-watcher-api = { path = "../../watcher/api" }
 
 # fog
+mc-fog-api = { path = "../api/" }
```

Comment:
This is probably going to break things because fog types goes in the enclave and mc-fog-api uses grpcio and cannot go in the enclave. I think the from/into trait impl should be in the mc-fog-api crate onstead